README.md

```diff
@@ -3,741 +3,470 @@
-    bytes32 qualificationTypehash; // Typehash of the qualification payload.
-    bytes qualificationPayload; // Data used to derive qualification hash.
-    // ... same last 5 parameters as MultichainClaim
-}
+**3. Obtain Allocator Authorization:**
+    - This relies on the allocator's on-chain `authorizeClaim` logic. Note that the arbiter may submit `allocatorData` (i.e., an allocator's signature or other proof the allocator understands) which the allocator can evaluate as part of its authorization flow.
 
-struct ExogenousQualifiedMultichainClaim {
-    // ... same first 5 parameters as all claims
-    bytes32 qualificationTypehash; // Typehash of the qualification payload.
-    bytes qualificationPayload; // Data used to derive qualification hash.
-    bytes32[] additionalChains; // The element hashes from additional chains.
-    uint256 chainIndex; // The index after which to insert the current element hash.
-    uint256 notarizedChainId; // The chain id used to sign the multichain claim.
-    // ... same last 4 parameters as MultichainClaim
-}
+**4. Submit the Claim:**
+    - Call the appropriate claim function on [`ITheCompactClaims`](./src/interfaces/ITheCompactClaims.sol) with the claim payload (e.g., `Claim`, `BatchClaim`).
+    - The payload includes `allocatorData`, `sponsorSignature` (if not registered), lock details, and `claimants` array.
+    - See [Key Concepts: Claimant Processing & Structure](#claimant-processing--structure) for how claims are processed.
+    - Successful execution emits a `Claim` event and consumes the nonce.
 
-struct MultichainClaimWithWitness {
-    // ... same first 5 parameters as all claims
-    bytes32 witness; // Hash of the witness data.
-    string witnessTypestring; // Witness typestring appended to existing typestring.
-    // ... same last 5 parameters as MultichainClaim
-}
+### Relayers
 
-struct ExogenousMultichainClaimWithWitness {
-    // ... same first 5 parameters as all claims
-    bytes32 witness; // Hash of the witness data.
-    string witnessTypestring; // Witness typestring appended to existing typestring.
-    bytes32[] additionalChains; // The element hashes from additional chains.
-    uint256 chainIndex; // The index after which to insert the current element hash.
-    uint256 notarizedChainId; // The chain id used to sign the multichain claim.
-    // ... same last 4 parameters as MultichainClaim
-}
+Relayers can perform certain interactions on behalf of sponsors and/or claimants.
 
-struct QualifiedMultichainClaimWithWitness {
-    // ... same first 5 parameters as all claims
-    bytes32 witness; // Hash of the witness data.
-    string witnessTypestring; // Witness typestring appended to existing typestring.
-    bytes32 qualificationTypehash; // Typehash of the qualification payload.
-    bytes qualificationPayload; // Data used to derive qualification hash.
-    // ... same last 5 parameters as MultichainClaim
-}
+**1. Relaying Permit2 Interactions:**
```

Comment:
There are also the batch versions of these; we should clarify as well that in the register variants the role is called the activator and is signed for by the sponsor as part of the permit2 witness data